java/src/main/java/ai/onnxruntime/providers/CoreMLFlags.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * Licensed under the MIT License.
+ */
+package ai.onnxruntime.providers;
+
+/** Flags for the CoreML provider. */
+public enum CoreMLFlags implements OrtFlags {
+  CPU_ONLY(1), // COREML_FLAG_USE_CPU_ONLY(0x001)
```

Comment:
>CPU_ONLY(1) [](start = 2, length = 11)

I wonder if EnumSet can be used for validation of the flags?

Reply:
What kind of validation?

Reply:
Valid flag values

---
In reply to: [581338623](https://github.com/microsoft/onnxruntime/pull/6770#discussion_r581338623) [](ancestors = 581338623)

Reply:
It only accepts valid enum constants of that type, and ensures that at most one of each enum constant is present in the set, so it's already doing that.